.github/workflows/backstop.yml

```diff
@@ -0,0 +1,123 @@
+name: Backstop Tests
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  run-backstop-tests:
+    if: github.event.label.name == 'run backstop'
+    runs-on: ubuntu-latest
+    container:
+      image: compucorp/civicrm-buildkit:1.0.0-chrome
+
+    env:
+      CIVICRM_EXTENSIONS_DIR: site/web/sites/all/modules/civicrm/tools/extensions
+      DRUPAL_THEME_DIR: site/web/sites/all/themes
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+        - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+      - name: Install dependencies to run the server
+        run: |
+          apt-get update
+          apt-get install apache2 libapache2-mod-php7.2 php7.2-mysql php7.2-curl php7.2-simplexml -y
+          a2enmod rewrite
+          apache2ctl restart
+
+      - name: Config mysql database as per CiviCRM requirement
+        run: echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" | mysql -u root --password=root --host=mysql
+
+      - name: Config amp
+        run : |
+          amp config:set --db_type=mysql_dsn --mysql_dsn='mysql://root:root@mysql:3306' --httpd_type=apache24 --httpd_restart_command='sudo /usr/sbin/apache2ctl graceful' --perm_type=worldWritable --hosts_type=file
+          echo "IncludeOptional /github/home/.amp/apache.d/*.conf" >> /etc/apache2/apache2.conf
+          /usr/sbin/apache2ctl restart
+
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.15.3'
```

Comment:
I was thinking, that the node version is updated sometimes and it is also possible to have different node versions in different projects. So I think its better to keep it here, so that every project can configure as per need.

Reply:
@deb1990 from what I've been seeing, this version is what is being used in most of our projects. If that is really the case, then it does make sense to make it the default one shipped with the container and only use something like `setup-node` in case one specific project needs something different